pkg/sources/v1alpha2/container_client.go

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"context"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	knerrors "knative.dev/client/pkg/errors"
+	"knative.dev/client/pkg/util"
+	v1alpha2 "knative.dev/eventing/pkg/apis/sources/v1alpha2"
+	"knative.dev/eventing/pkg/client/clientset/versioned/scheme"
+	clientv1alpha2 "knative.dev/eventing/pkg/client/clientset/versioned/typed/sources/v1alpha2"
```

Comment:
I wonder if we can aim for `v1` here rather than `v1alpha2`, since it's available on current Eventing `master`. Therefore it will be in `0.19` release.

https://github.com/knative/eventing/blob/master/pkg/apis/sources/v1/container_types.go

Reply:
Good suggestion. I will suggest we upgrade all build-in sources (ApiServerSource, SinkBinding and ContainerSource) from v1alpha2 to v1 in another PR.
I don't see much difference in container sources between v1alpha2 and v1. We need to check the spec for other two build-in sources. Agree ?

Reply:
I have had a similar question to Navid that he actually replied to that it's fine to use older API as we want to support it until it goes away.

/cc @navidshaikh am I right here? :)

Reply:
I created an issue https://github.com/knative/client/issues/1160. Let's do the upgrade in following PRs. Currently, Kn Client support only 1 version of Eventing, that is v1alpha2. We might not want to support v1alpha2 of apiserversource, sinkbinding and pingsource, while another version of container source.

Reply:
Yeah, we are following the "defensive" approach as described in https://github.com/knative/client/issues/462